README.md

```diff
@@ -424,15 +445,28 @@ public class LoggingEventErrorHandler implements EventErrorHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(LoggingEventErrorHandler.class);
 
     @Override
-    public void onError(final Throwable t, final EventTypePartition eventTypePartition, @Nullable final String offset,
+    public void onError(final String consumerName, final Throwable t, final EventTypePartition eventTypePartition, @Nullable final String offset,
             final String rawEvent) {
         LOGGER.error(
-            "Failed Event // Event Partition = [{}] , Event Type = [{}] , Event Offset = [{}] , Raw Event = [{}] //",
-            eventTypePartition.getPartition(), eventTypePartition.getEventType(), offset, rawEvent, t);
+            "Failed Event // Consumer Name = [{}] , Event Partition = [{}] , Event Type = [{}] , Event Offset = [{}] , Raw Event = [{}] //",
+            consumerName, eventTypePartition.getPartition(), eventTypePartition.getEventType(), offset, rawEvent, t);
     }
 }
 ```
 
+If the sqs error handler is enabled, it will store the failed events to the sqs queue automatically.
+
+Here are the configuration explanations 
+
+```properties
+paradox.nakadi.errorhandler.sqs.enabled : If it is true, the handler is binded to context automatically.
+paradox.nakadi.errorhandler.sqs.queueName: The place where the failed events are stored.
+paradox.nakadi.errorhandler.sqs.region: AWS region
+paradox.nakadi.errorhandler.sqs.createQueueIfNotExists: If the parameter is true, initially, it checks the queue name in amazon then creates it if it does not exist.
+paradox.nakadi.errorhandler.sqs.messageVisibilityTimeout: The length of time (in seconds) that a message received from a queue will be invisible to other receiving components. The value must be between 0 and 43200.
+paradox.nakadi.errorhandler.sqs.messageRetentionPeriod: The amount of time that Amazon SQS will retain a message if it does not get deleted. The value must be between 60 and 1209600.
```

Comment:
Units are missing (seconds). Description could be outside of the `monospaced` section. Reference to http://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SetQueueAttributes.html wouldn't hurt.